Sources/Containerization/LinuxProcess.swift

```diff
@@ -335,16 +354,50 @@ extension LinuxProcess {
         }
     }
 
+    /// Wait until the standard output and standard error streams for the process have concluded.
+    private func waitIoComplete() async throws {
+        let ioTracker = self.state.withLock { $0.ioTracker }
+        guard let ioTracker else {
+            return
+        }
+        let required = {
```

Comment:
nit: We might be able to do this dance before creating the object and just pass directly to the object itself how many streams we need to wait for. The guard below (although honestly maybe being defensive here is good, but it seems a bit much) seems unlikely to ever be hit, as we only create the tracker if we asked for stdout/stderr